docs/user/gtfs-plus-editor.md

```diff
@@ -33,3 +33,59 @@ After making changes to the data, the "Save and Revalidate" button can be used t
 
 Once the GTFS+ data for a given Feed Version is to a point where it is complete and ready for external use, it may be exported as a new Feed Version in the main Data Manager using the "Create a New Version" button within the GTFS+ summary panel (note that this button does not appear until the GTFS+ for a given Feed Version has been edited and saved at least once). In addition to the updated GTFS+ data, any standard GTFS data from the original Feed Version will be retained and included in the newly created version.
 ![publish GTFS+ as new version](../img/gtfsplus-publish.png)
+
+## Editing `route_attributes.txt`
+
+### Opening the GTFS+ Editor
+
+From the Feed Version viewer, clicking the "GTFS+ for this version" shortcut on the left side of the screen will open the GTFS+ summary page that lists GTFS+ files and validation summary.
+
+If editing `route_attributes.txt` from scratch, the row for `route_attributes.txt` will be grayed out in the GTFS+ summary page. Click "Edit GTFS+" to the top-right corner to open the GTFS+ Editor.
+
+Note that route attributes can only be added for the transit routes defined for the active agency feed. To edit route attributes for another agency, navigate back to the applicable project and select the desired agency and feed version.
+
+### Route Attributes Editor Overview
+
+Once in the GTFS+ editor, select "route_attributes" from the GTFS+ files shown on the left side. "route attributes" will be grayed-out but still available for editing if no data were previously entered.
```

Comment:
Add underscore to `route attributes`. I think.

Reply:
Good find, thanks! Fixed in https://github.com/ibi-group/datatools-ui/pull/796/commits/41e95f9eadc582017c801d7b6d0ad7ac5843aeff.